omx/gstomxaudiodec.c

```diff
@@ -279,9 +274,6 @@ gst_omx_audio_dec_change_state (GstElement * element, GstStateChange transition)
     case GST_STATE_CHANGE_PAUSED_TO_READY:
       self->downstream_flow_ret = GST_FLOW_FLUSHING;
       self->started = FALSE;
-
-      if (!gst_omx_audio_dec_shutdown (self))
```

Comment:
Why is this no longer required?

Reply:
This is call was a duplicate, the class has the _close call that gets in charge of calling the _shutdown function. The _close function is already registered in the _class_init. This is also part of an element tear down fix.